### PR TITLE
Add direct link to auto discovery docs in mysql/postgres aurora setup

### PR DESCRIPTION
**Pull request by jmeunier28**
### What does this PR do? What is the motivation?

Adds direct note in setup docs on how to use the new auto discovery feature in the agent

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

File: content/en/database_monitoring/setup_postgres/aurora.md [hunk 1]
```diff
@@ -220,6 +220,8 @@ To configure collecting Database Monitoring metrics for an Agent running on a ho
 
 2. [Restart the Agent][2].
 
+**The Datadog Agent supports auto-discovering all aurora endpoints in a cluster.** In order to do this, please [follow this guide.][14]
```
Comment (aliciascott) [inline content/en/database_monitoring/setup_postgres/aurora.md:223]:
```suggestion
**The Datadog Agent supports [Autodiscovery][14] of all Aurora endpoints in a cluster.** 
```

Same with this one

File: content/en/database_monitoring/setup_mysql/aurora.md [hunk 2]
```diff
@@ -198,6 +198,8 @@ instances:
 
 [Restart the Agent][3] to start sending MySQL metrics to Datadog.
 
+**The Datadog Agent supports auto-discovering all aurora endpoints in a cluster.** In order to do this, please [follow this guide.][10]
```
Comment (aliciascott) [inline content/en/database_monitoring/setup_mysql/aurora.md:201]:
```suggestion
**The Datadog Agent supports [Autodiscovery][10] of all Aurora endpoints in a cluster.** 
```

Simplifying this sentence so it flows better to just link directly within to the autodiscovery config page (also the link wasn't working as is).

**Review by aliciascott — CHANGES_REQUESTED** [review]
Hi @jmeunier28 this looks good, just some small suggestions, please request re-review once updated, thanks!

**Comment by jmeunier28** [comment]
@aliciascott  thanks!

**Review by aliciascott — APPROVED** [review]
Hi @jmeunier28 I forgot to mention that the links that are within tabs need to be placed within the tab section, I just pushed up a fix to correct that, let me know if this is ready to be merged as well?

**Comment by jmeunier28** [comment]
Ah sorry about that! Yes @aliciascott please merge whenever you get a chance! Thanks so much!

**Pull request merged**